config/routes.rb

```diff
@@ -3,11 +3,11 @@
 
   # Defines the root path route ("/")
   root "rails/welcome#index"
-  resources :users
   resources :questions do
     resources :answers, only: [:index], controller: "questions/answers"
   end
   resources :answers
+  resources :friends, only: [:show], controller: "weekly_friends"
```

Comment:
```suggestion
  resources :weekly_friends, only: [:show], controller: "weekly_friends"
```

Why not this? Is the URL being only `friends` important?

Reply:
It's not mandatory, it can absolutely be changed. It's just easier to write/read.